root/static/styles/favicons.less

```diff
@@ -34,6 +34,7 @@
 .favicon("genius");
 .favicon("googleplus", 32);
 .favicon("ibdb");
+.favicon("IMDb");
```

Comment:
Also, all the others are all lowercase, so this and VGMdb should probably be as well.